llama_stack/providers/inline/post_training/torchtune/recipes/lora_finetuning_single_device.py

```diff
@@ -484,7 +487,7 @@ async def train(self) -> Tuple[Dict[str, Any], List[Checkpoint]]:
                 # Loss is normalized by default so we multiply by the number of tokens
                 # This way we can normalize by the total number of tokens if we're accumulating gradients
                 current_loss = await self._loss_step(batch) * current_num_tokens
-                running_loss += current_loss
+                running_loss += current_loss.detach().item()
```

Comment:
nit: you don't need the `item()` right? because you are converting it back to this not being a tensor? why make this a tensor in the first place though?

Reply:
Yeah I guess that's true- `.item()` returns a float so it can just be `running_loss: float = 0.0`

Reply:
Indeed!